src/components/sidePanelDocs/Iframe.tsx

```diff
@@ -67,12 +72,11 @@ function SidePanelIframe() {
                 setLoading(false);
             }, 100);
         };
-        iframeCurrent?.addEventListener('load', hideNavBar);
 
-        return () => {
-            iframeCurrent?.removeEventListener('load', hideNavBar);
-        };
-    }, [iframeCurrent, setAnimateOpening]);
+        if (iframeCurrent && show && hasLength(docsURL)) {
+            hideNavBar();
```

Comment:
If we only call this within this block I think we should just move the declaration inside as well.

Reply:
Definitely. I wanted to make sure the code was progressing in the right direction before removing the function definition outright.